CONTRIBUTING.md

```diff
@@ -49,6 +49,13 @@ To make a change, update the corresponding file within the `data` folder: The `p
 
 Icons from different sources (_icon sets_) can be used in the tagging schema. Head over to the [dedicated page](https://github.com/ideditor/schema-builder/blob/main/ICONS.md#icons) about how to use them.
 
+### Info-`i`
+
+iD and other tools provide users with a way to learn more about the main tag of a preset. It is important to provide good information in this information panel. Here are a few notes on how to do this:
+- Does your tag have a Wikidata entry? Click the small pencil icon next to the text to open the Wikidata item on the OSM wiki. Improve this wording if needed. If the Wikidata item is missing, [learn more about how to add it in "Current methods for creating new items"](https://wiki.openstreetmap.org/wiki/Data_items#Item_creation_process).
```

Comment:
I think it might be helpful to mention where the "information panel" is, before saying _"Click the small pencil icon next to the text"_. 

For someone who is not familiar with data items, it might not be clear what this paragraph is about. A [screenshot like this](https://github.com/openstreetmap/id-tagging-schema/assets/111561/13549318-cd7c-4dd1-9948-7a2d84662f04) would also help, but it might look out of place, since there are no other images in this document.

Reply:
(Is this screenshot URL permanent / can I use it in the readme?)

Reply:
l grabbed that URL from your earlier comment, and I believe GitHub preserves all uploaded images if they're used in at least one comment

Reply:
Thanks. I added the image. It is a bit too big and also not ideal from what it shows. But we can update that later IMO, once we know this link will work mid term… 
Lets again in a few days.